recipes/octo-logger-cpp/all/conanfile.py

```diff
@@ -28,6 +34,14 @@ def _compilers_minimum_version(self):
             "msvc": "1923",
         }
 
+    @property
+    def _aws_supported(self):
+        return Version(self.version) >= "1.2.0"
+
+    def configure(self):
+        if self.options.with_aws and self._aws_supported:
+            self.options["aws-sdk-cpp"].logs = True
```

Comment:
```suggestion
            self.options["aws-sdk-cpp"].logs = True
        if not self._aws_supported:
            del self.options.with_aws
```
Older version should not support it.